jsonsl.c

```diff
@@ -378,7 +391,8 @@ jsonsl_feed(jsonsl_t jsn, const jsonsl_char_t *bytes, size_t nbytes)
                 goto GT_SPECIAL_NUMERIC;
             }
 
-            if (state->special_flags & JSONSL_SPECIALf_NUMERIC) {
+            if (state->special_flags & JSONSL_SPECIALf_NUMERIC &&
```

Comment:
I'd prefer some `(parens)`.

Also, maybe ifdef this out too? - I'm guessing any compiler worth its salt should be able to optimize this out, though .. -- eg. foo&bar && !&foo&0